howto/troubleshoot/dashboard-issues.md

```diff
@@ -0,0 +1,65 @@
+## Session with an Error status
+*Applies to: Anbox Cloud, Anbox Cloud Appliance*
+
+On the **Sessions** page, you could see a session with an **Error** status when there are not enough resources to start a streaming session. 
+
+![Session with an Error status|690x440](https://assets.ubuntu.com/v1/91739759-session-error.png)
+
+Try the following actions:
+* Verify if you have [sufficient resources](https://discourse.ubuntu.com/t/capacity-planning/28717) for container/application creation.
+* Check if all the nodes are in [`unschedulable`](https://discourse.ubuntu.com/t/ams-configuration/20872) mode.
+
+## Session does not start
+*Applies to: Anbox Cloud, Anbox Cloud Appliance*
+
+A session does not start and the session details page displays the following error:
+
+![Session does not start|690x440](https://assets.ubuntu.com/v1/52b32f73-session-does-not-start.png)
+
+[Check the container logs](https://discourse.ubuntu.com/t/how-to-view-the-container-logs/24329) to find reasons for the session failure.
+
+
+## Container(s) in Error status
+*Applies to: Anbox Cloud, Anbox Cloud Appliance*
+
+The **Containers** list page shows containers with Error status.
+
+![Container in Error status|690x440](https://assets.ubuntu.com/v1/aaf1194c-container-list-error.png)
+
+A container can end up with an error status due to various reasons. It may not always be simple and easy to resolve this because of the variable factors involved, for example, the application that the container is hosting or any installed addons.
+
+Click on the corresponding **CONTAINER ID** and check the container details page for any possible error messages. The container details page has an **Error Message** field that could be useful.
+
+![Container details page|690x440](https://assets.ubuntu.com/v1/0ff0d3ff-container-details-error.png)
+
+The **Error Message field** can give you a starting point for identifying the issue. Some reasons for a container to go into error status could be:
+* Insufficient resources. Refer [Capacity planning](https://discourse.ubuntu.com/t/capacity-planning/28717).
+* Occasionally, access to Ubuntu archives could be a problem when creating an application. As an immediate workaround, you could disable the security update by running `amc config set container.security_updates false` or explicitly set `amc config set container.api_mirror <mirror_address>`.
```

Comment:
or explicitly set `amc config set container.api_mirror <mirror_address>`.
--->
 or explicitly set `amc config set container.apt_mirror <mirror_address>` to configure a container to use a different apt mirror.